dev-tools/docker/linux_image/Dockerfile

```diff
@@ -33,8 +33,11 @@ RUN \
 ENV LD_LIBRARY_PATH /usr/local/gcc73/lib64:/usr/local/gcc73/lib:/usr/lib:/lib
 ENV PATH /usr/local/gcc73/bin:/usr/bin:/bin:/usr/sbin:/sbin
 
-# For compiling in C++14 mode
+# For compiling in C++14 mode with hardening and optimisation
```

Comment:
Note that optimisation was _not_ added in this commit.  If `CFLAGS` is not specified then the `configure` scripts use a default value for it, and this invariably includes optimisation.  It's only because we are now specifying `CFLAGS` that we have to explicitly add the optimisation option now.